our-brothers-web/src/app/shared/components/free-text-filter/free-text-filter.component.scss

```diff
@@ -5,12 +5,20 @@
   background-color: rgb(0, 160, 153);
   box-shadow: 0px 1px 10px 0px rgba(1, 1, 1, 0.15);
 
+  @media only screen and (max-width: 768px) {
+    border: 7px solid rgb(0, 160, 153);
```

Comment:
`border-width: 7px;`

Reply:
I tried, it does not give the same response